### PR TITLE
MAIN-9881: Upgraded PHPUnit to 6.0.*

### PR DESCRIPTION
**Pull request by mszabo-wikia**
https://wikia-inc.atlassian.net/browse/MAIN-9881

### CODE REVIEW

File: extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
-class ScavengerHuntTest extends WikiaBaseTest {
+abstract class ScavengerHuntTest extends WikiaBaseTest {
```
Comment (macbre) [inline extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php:2]: Worth checking if we actually have ` ScavengerHunt` enabled anywhere.

File: lib/Wikia/tests/Logger/WikiaLoggerTest.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
 use Wikia\Logger\SyslogHandler;
 use Wikia\Logger\WebProcessor;
 
-class WikiaLoggerTest extends PHPUnit_Framework_TestCase {
+class WikiaLoggerTest extends \PHPUnit\Framework\TestCase {
```
Comment (macbre) [inline lib/Wikia/tests/Logger/WikiaLoggerTest.php]: Let's be consistent when extending `TestCase` class, either provide the full "path" with namespaces or use `use`.

File: extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php (hunk 1 above)
Comment (mszabo-wikia) [reply extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php:2]: Yeah. All tests were marked as `Broken` 4 years ago and the extension might be disabled now.

**Review by macbre — CHANGES_REQUESTED** [review]

File: extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php:2]: https://wikia-inc.atlassian.net/browse/SUS-1730

**Comment by macbre** [comment]
:+1: let's make the `TestCase` use consistent and we're good :)

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
@TK-999, worth checking why `unitTests_php_branches` Jenkins task did not run (or at least did not report the results here).

**Comment by macbre** [comment]
:+1:  if tests do pass now, let's simply merge it and announce the good news

Thanks, @TK-999  :)

**Pull request merged**